src/world/area_kmr/kmr_20/8EBE50.c

```diff
@@ -2,11 +2,20 @@
 
 #include "world/common/atomic/Pipe.inc.c"
 
-INCLUDE_ASM(s32, "world/area_kmr/kmr_20/8EBE50", func_80240310_8EC130);
+ func_80240310_8EC130(void) {
+    set_map_transition_effect(4);
+    return 2;
+}
```

Comment:
Please make this change to all 4 funcs
```suggestion
ApiStatus func_80240310_8EC130(Evt* script, s32 isInitialCall) {
    set_map_transition_effect(4);
    return ApiStatus_DONE2;
}
```